src/common/chat-participants/powerpages/PowerPagesChatParticipantUtils.ts

```diff
@@ -28,7 +30,7 @@ export async function getEndpoint(
     * Get component info for the active file
     * @returns componentInfo - Entity details for active file (form or list)
 */
-export async function getComponentInfo(telemetry: ITelemetry, orgUrl: string | undefined, activeFileParams: IActiveFileParams): Promise<string[]> {
+export async function getComponentInfo(telemetry: ITelemetry, orgUrl: string | undefined, activeFileParams: IActiveFileParams): Promise<{ componentInfo: string[], entityName: string }> {
```

Comment:
can be converted into an interface:
<{ componentInfo: string[], entityName: string }